### DISCUSSION TITLE
End-to-end and cable connections partly redundant

### DISCUSSION THREAD

**Discussion by dreng** [mattieserver/netbox-topology-views#245] [Q&A]
### NetBox version

v3.4.3-dev

### Topology Views version

v3.2.0

### Steps to Reproduce

1. Navigate to "Topology Views" => "Topology"
2. In the "Filters" tab, check at least "Show Cables" and "Show Logical Connections"
3. Add some more filters (optional), include at least switches and patch panels
4. Have a look at the following screenshot. Notice redundant connections between switches (1). No redundancy is displayed between patch panels and switches (2). Both examples have got directly connected cables.
![site_topo_001](https://user-images.githubusercontent.com/20901110/211114804-42651220-58d1-4f14-a151-fefd7e03709e.png)

### Expected Behavior

No redundancy at all as this should have been addressed in commit 840044f.

### Observed Behavior

See above.

**Comment by dreng** [comment]
@tgrusendorf I'd like to assign this issue to you. Would you mind to take work on it again?

**Comment by tgrusendorf** [comment]
@dreng, sure.
What is your current value for the 'hide_single_cable_logical_conns' plugin config setting?

**Comment by dreng** [comment]
@tgrusendorf Oh my gosh, this is embarassing. I didn't remember that I switched to another venv and a different NetBox folder.

Going to convert this into a discussion since it's not a bug but a technical question.

However, I am thinking about revising the options in general. I find it inconvenient and not very user-friendly to map some options in the filter and others in the config file. Maybe it would be better to manage everything in the database. If I remember correctly, there is already a similar feature request.

**Reply by tgrusendorf** [reply]
@dreng, no worries!

I agree it's not as user friendly as I would like.  I'd like to see it as a setting that can be changed from the GUI but is maintained 'per-user' (just like table layouts).  I'd kind of like dash style to be a user preference too as I prefer the solid lines most of the time.  I'm not sure what options exist for maintaining user preferences related to plugins.

Is #78 the feature request you're thinking of?

**Reply by dreng** [reply]
> Is #78 the feature request you're thinking of?

Correct.
